### DISCUSSION TITLE
Remove previous revisions from kubernetes cluster

### DISCUSSION THREAD

**Discussion by swapkh91** [kubeflow/kfserving#2302] [General]
![Screenshot from 2022-07-01 16-46-41](https://user-images.githubusercontent.com/5755405/176885022-33997716-e1e4-47a2-8279-34eec84320a8.png)

I want to remove the 2 outdated revisions 00001 and 00002 from the list. As the number of iterations increase for a deployment, the workloads list on GKE becomes crowded.

I tried manually removing these from the GKE UI but they just get re-created with 0/0 pods. Running `kubectl delete deployment <revision_name>` does the same.

Running `kubectl delete` removes all the revisions.

@yuzisun

**Comment by yuzisun** [comment]
Knative revision GC should clean up these automatically, see https://knative.dev/docs/serving/configuration/revision-gc. If you want to manually clean up you can run `kubectl delete revision <revision_name>`

**Reply by swapkh91** [reply]
@yuzisun thanks, that worked. Had to patch the configmap.
